src/constants.cpp

```diff
@@ -117,7 +118,7 @@ const wxString mmex::getProgramDescription()
         << bull + LUA_RELEASE << "\n"
         << bull + curl << "\n\n"
 
-        << wxString::Format(_("Build on %s %s with:"), __DATE__, __TIME__) << "\n"
+        << _("Build on") << " " << __DATE__ << " " << __TIME__  << " " << _("with:") << "\n"
```

Comment:
It's bad for translators. What does "with" mean?

Reply:
It could be simpler if i change this function to `const wxString mmex::getProgramDescription(bool translate = true)`...